hail/python/hail/docs/change_log.md

```diff
@@ -1,5 +1,37 @@
 # Change Log
 
+## 0.2.17
+
+Released 2019-07-10
+
+### New features
+
+- (hail#6349) Added `compression` parameter to `export_block_matrices`, which can
+  be `'gz'` or `'bgz'`.
+- (hail#6405) When a matrix table has string column-keys, `matrixtable.show` uses
+  the column key as the column name.
+- (hail#6345) Added an improved scan implementation, which reduces the memory
+  load on master.
+- (hail#6462) Added `export_bgen` method.
+
+### Bug fixes
+
+- (hail#6404) `n_rows` and `n_cols` parameters added to `Expression.show` for
+  consistency with other `show` methods.
+- (hail#6408)(hail#6419) Fixed an issue where the `filter_intervals` optimization
+  could make scans return incorrect results.
+- (hail#6459)(hail#6458) Fixed rare correctness bug in the `filter_intervals`
+  optimization which could result too many rows being kept.
+- (hail#6496) Fixed html output of `show` methods to truncate long field
+  contents.
+- (hail#6478) Fixed the broken documentation for the experimental `approx_cdf`
+  and `approx_quantiles` aggregators.
+
```

Comment:
(hail#6504) Fix `Table.show` collecting data twice while running in Jupyter notebooks.
(hail#6571) Fixed the message printed in `hl.concordance` to print the number of overlapping samples, not the full list of overlapping sample IDs.

Reply:
also, just went in:

(hail#6583) Fixed `hl.plot.manhattan` for non-GRCh37 reference genomes.